addons/l10n_it_edi/models/res_config_settings.py

```diff
@@ -19,9 +19,9 @@ class ResConfigSettings(models.TransientModel):
         readonly=False)
 
     def _create_proxy_user(self, company_id, edi_mode):
-        fattura_pa = self.env.ref('l10n_it_edi.edi_fatturaPA')
-        edi_identification = fattura_pa._get_proxy_identification(company_id)
-        self.env['account_edi_proxy_client.user']._register_proxy_user(company_id, 'l10n_it_edi', edi_mode, edi_identification)
+        proxy_user = self.env['account_edi_proxy_client.user']
+        edi_identification = proxy_user._get_proxy_identification(company_id)
```

Comment:
There's a problem with `_get_proxy_identification()` because both `account_peppol` and `l10n_it_edi` override it, so if both modules are installed you can't know which override will be called.

Reply:
👍🏻